### DISCUSSION TITLE
M1 Mac UVTools 3.7.2

### DISCUSSION THREAD

**Discussion by SmartDucksWorks** [sn4k3/UVtools#595] [General]
Installed, but fails to run, 

UVtools v3.7.2 Arm64 [macOS]
Unable to run due one or more missing dependencies.
Triggered by: libcvextern  (OpenCV)

Looking for dependency to install.... suggestions?

**Comment by sn4k3** [comment]
I recommend you to wait for the new release which will be out today.
Or use the x64 package instead

**Comment by SmartDucksWorks** [comment]
It's that ffmpeg issue, got it working after I chased down a couple rabbit holes.  I'd tell you what to do with x64 but I hardly know you. ;)

**Reply by sn4k3** [reply]
It's nothing about ffmpeg, please read the links i posted on your open issue some mins ago

**Comment by sn4k3** [comment]
You can now install by follow: https://github.com/sn4k3/UVtools#to-auto-install-on-macos-homebrew

**Reply by SmartDucksWorks** [reply]
Yup. Issue resolved. Appreciated.
